SocialCareCaseViewerApi.Tests/V1/Factories/EntityFactoryTests.cs

```diff
@@ -378,7 +379,7 @@ public void CanMapPhoneNumberFromDomainToInfrastructure()
 
             var infrastructurePhoneNumber = new dbPhoneNumber()
             {
-                Number = phoneNumber.ToString(),
+                Number = phoneNumber,
```

Comment:
Unrelated to the actual purpose of the PR but no need to call `.ToString()` against something that is already type `String`